crmd/notify.c

```diff
@@ -20,120 +20,501 @@
 #include <crm/crm.h>
 #include <crm/msg_xml.h>
 #include "notify.h"
+#include "crmd_messages.h"
 
 char *notify_script = NULL;
 char *notify_target = NULL;
+GListPtr notify_list = NULL;
+
+typedef struct {
+    char *id;
+    char *path;
+    int timeout;
+    char *tstamp_format;
+    char *info;
```

Comment:
We support a list of recipients so I thought a field common for all recipients of one script might be handy. In terms of email it could for instance be a subject/part of a subject, a mail-domain, ...
Regarding being used for descriptive purposes I of course thought of XML comments as well but I haven't seen them being used by high-level-tooling  like pcs and it might be handy to have them at hand in the script as well to do some local logging for instance.